Dart/src/com/jetbrains/lang/dart/ide/errorTreeView/DartFeedbackBuilder.java

```diff
@@ -38,40 +42,61 @@ default String title() {
    *
    * @return The string to display in the confirmation dialog.
    */
-  String prompt();
+  public abstract String prompt();
 
   /**
    * The label should indicate what is going to happen when clicked (eg send feedback).
    *
    * @return The string to display as the label of the yes-button in the confirmation dialog.
    */
-  default String label() {
+  public String label() {
     return "Send feedback";
   }
 
   /**
-   * Set the text of the message to be optionally included in the report.
-   *
-   * @param message
-   */
-  default void setMessage(String message) {
-  }
-
-  /**
    * Perform the action required to send feedback.
    *
    * @param project the current project
+   * @param errorMessage additional information for the issue, such as a tack trace
    */
-  void sendFeedback(@Nullable Project project);
+  public abstract void sendFeedback(@Nullable Project project, String errorMessage);
```

Comment:
I've added `@Nullable` to the `errorMessage` param here and for the implementation